src/logic/svgRenderer.ts

```diff
@@ -10,22 +10,67 @@ import { FontStyle, IThemedToken } from 'shiki';
 import type { HTMLEscapes, RendererOptions, SVGAttributes, SVGOutput } from '../types/renderer';
 
 const DEFAULT_CONFIG: Partial<RendererOptions> = {
-  fontFamily: 'JetBrainsMono Nerd Font',
+  fontFamily: 'JetBrains Mono',
```

Comment:
ini kenapa diubah ke non-nerd font?

I mean it's ok but, ga mau sekalian ganti di `src/logic/genFont.ts` ? terus harus tuker yang ada di folder fonts

Reply:
sek, emang yg di folder itu versi nerd font? aku ganti soalnya ini sisaan pake font system, punyaku nerdfont

Reply:
well of course. we both use starship arent we lmao